src/components/cc-toggle/cc-toggle.js

```diff
@@ -174,40 +176,21 @@ export class CcToggle extends LitElement {
 
           display: inline-flex;
         }
-
-        /* RESET */
-
-        fieldset {
-          display: inline-block;
-          min-width: 0;
-          padding: 0;
-          border: 0;
-          margin: 0;
-        }
-
-        /* RESET */
-
-        legend {
-          max-width: 100%;
-          padding: 0;
-          color: inherit;
-          line-height: inherit;
-          white-space: normal;
+        
+        .group {
+          display: flex;
+          flex-direction: column;
+          gap: 0.35em;
         }
 
-        legend:not(:empty) {
-          padding-bottom: 0.35em;
-          line-height: 1.25em;
+        :host([inline]) .group {
+          flex-direction: row;
+          align-items: center;
+          gap: 1em;
         }
-
-        :host([inline]) legend {
-          width: max-content;
-          padding: 0;
-          margin-right: 1em;
-          /* cannot use flex to change legend position. Only solution is float. */
-          float: left;
-          /* used to vertically center the floated element. */
-          line-height: var(--height);
+        
+        #legend {
+          height: 1.25em;
```

Comment:
**suggestion:** I think you want `line-height` here instead so that it's exactly the same look as our labels in `cc-input`. 

This is a very minor issue and it's actually quite annoying to maintain this kind of stuff, we'll see how we can improve that when we decide on a way to share styles between form components.